set.mm

```diff
@@ -525783,6 +525786,108 @@ FOL part ( ~ bj-ru0 ) and then two versions ( ~ bj-ru1 and ~ bj-ru ).
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Nonfreeness
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $( Token for the nonfreeness quantifier. $)
+  $c F// $.
+
+  $( Syntax for the nonfreeness quantifier. $)
+  wnnf $a wff F// x ph $.
+
+  $( Definition of the nonfreeness quantifier.  Compared with ~ df-nf , the
+     present definition is stricter on minimal propositional calculus
+     ( ~ bj-nnfnfTEMP ) and equivalent on FOL ( ~ bj-nfnnfTEMP ).  While being
+     stricter, it still holds for non-occurring variables ( ~ bj-nnfv ), which
+     is the basic requirement for this predicate.  In particular, it translates
+     more closely the associated disjointness condition.  Since the nonfreeness
+     quantifier is a means to translate a disjointness condition from the
+     metalogic to the logic, it seems preferable.  Also, since nonfreeness is
+     mainly used as a hypothesis, this definition would allow more theorems,
+     notably the 19.xx theorems, to be proved from the core axioms, without
+     needing a 19.xxv variant.
+
+     One can devise infinitely many definitions increasingly close to the
+     non-occurring condition, like
+     ` ( ( E. x ph -> ph ) /\ ( ph -> A. x ph ) ) /\ `
+     ` A. x ( ( E. x ph -> ph ) /\ ( ph -> A. x ph ) ) /\ A. x A. x ` ... and
+     each stronger definition would permit more theorems to be proved from the
+     core axioms.  A reasonable rule seems to be to stop at nested quantifiers
+     (on the other hand, going one more step would allow to prove virtually all
+     the theorems we currently have...).  Nested quantifiers over a common
+     variable are primarily dealt with by ~ ax-10 .  (Contributed by BJ,
+     28-Jul-2023.) $)
+  df-bj-nnf $a |-
```

Comment:
The current definiens is `( E. x ph -> A. x ph )` and the previous definiens was `A. x ( ph -> A. x ph )`. The current one was already an improvement (looking at axiom usage in  theorems using it), in particular because it avoided nested quantifiers. The one I propose is  `( ( E. x ph -> ph ) /\ ( ph -> A. x ph ) )` so it is obviously stronger than the current one, has no nested quantifiers (which typically require ax-10 to work with), and still holds when DV(x,ph) (which is the basic requirement). Adopting it would reduce axiom usage in more theorems. It is not ready yet; I want to check a few things and add a few theorems to illustrate that fact.